ee/tabby-webserver/src/repositories/mod.rs

```diff
@@ -62,8 +67,11 @@ async fn not_found() -> StatusCode {
 
 #[instrument(skip(repo))]
 async fn resolve(Path(repo): Path<ResolveParams>) -> Result<Response, StatusCode> {
-    let root = repositories_dir().join(repo.name_str());
-    let full_path = root.join(repo.path_str());
+    let Some(conf) = CONF.find_repository(repo.name_str()) else {
```

Comment:
instead of using a global variable, Conf can be passed in `attach_webserver` and being formed as a state here.